plugins/woocommerce/includes/class-woocommerce.php

```diff
@@ -402,6 +403,9 @@ public function log_errors() {
 				$context
 			);
 
+			$remote_logger = wc_get_container()->get( RemoteLogger::class );
```

Comment:
It doesn't seem like the remote logger has been registered, unless I'm missing something, so whenever execution reaches this point things fail with a new exception:

`Alias (Automattic\WooCommerce\Loggers\RemoteLogger) is not being managed by the container or delegates`

Are there some uncommitted changes we need?

Reply:
Sorry. I updated the namespace upstream but forgot to update it in this branch. I'll fix it.

Reply:
Fixed in e4e3019a02b53ee72a1c159f2ce3c1f66bc7c502.